core/src/main/java/org/apache/spark/h2o/JavaH2OContext.java

```diff
@@ -45,16 +48,16 @@
  * can be set in H2O configuration class H2OConf
  *
  */
-public class JavaH2OContext {
+public class JavaH2OContext implements Serializable {
```

Comment:
It was changed just recently when I was debugging one of the problems and it complained that `JavaH2OContext` is not serializable. We already have `H2OContext` implementing serialisable but having `sparkContext` as `@transient` field and that's why I decided to make this class also implement `Serializable` too even though I thought it's not a good idea.

I'm absolutely up for it having it non-serializable. It makes sense - does it mean that `H2OContext` should be changes as well ?

Thanks for pointing this out!